scripts/build_dists_zip.sh

```diff
@@ -199,53 +203,77 @@ echo "Hash file $hashFile"
 else
-  echo "Temporary directory $distDir does not exist"
+  echo "Temporary directory ${distDir} does not exist"
   exit 6
 fi
 
-mkdir $distDir/dist
-
-COPY_FILES=(
-  "target/release/tari_base_node"
-  "target/release/tari_console_wallet"
-  "target/release/tari_merge_mining_proxy"
-  "target/release/tari_mining_node"
-  "common/config/presets/tari_sample.toml"
-  "common/config/tari_config_example.toml"
-  "common/logging/log4rs_sample_base_node.yml"
-  "applications/tari_base_node/README.md"
-  applications/tari_base_node/$osname/*
-  "scripts/install_tor.sh"
-)
-
-for COPY_FILE in "${COPY_FILES[@]}"; do
-  cp -vr "$COPY_FILE" "$distDir/dist/"
-done
-
-pushd $distDir/dist
+# One click miner
+cp -f -P "${PWD}/applications/tari_base_node/osx/start_all" "${distDir}/start_all"
+cp -f "${PWD}/applications/tari_base_node/osx/runtime/start_all.sh" "${distDir}/runtime/start_all.sh"
+
+# Base Node
+cp -f -P "${PWD}/applications/tari_base_node/osx/start_tari_base_node" "${distDir}/start_tari_base_node"
+cp -f -P "${PWD}/applications/tari_base_node/osx/start_tor" "${distDir}/start_tor"
+cp -f "${PWD}/applications/tari_base_node/osx/runtime/start_tari_base_node.sh" "${distDir}/runtime/start_tari_base_node.sh"
+cp -f "${PWD}/applications/tari_base_node/osx/runtime/start_tor.sh" "${distDir}/runtime/start_tor.sh"
+cp -f "${PWD}/target/release/tari_base_node" "${distDir}/runtime/tari_base_node"
+
+# Console Wallet
+cp -f -P "${PWD}/applications/tari_console_wallet/osx/start_tari_console_wallet" "${distDir}/start_tari_console_wallet"
+cp -f "${PWD}/applications/tari_console_wallet/osx/runtime/start_tari_console_wallet.sh" "${distDir}/runtime/start_tari_console_wallet.sh"
+cp -f "${PWD}/target/release/tari_console_wallet" "${distDir}/runtime/tari_console_wallet"
+
+# Mining Node
+cp -f -P "${PWD}/applications/tari_mining_node/osx/start_tari_mining_node" "${distDir}/start_tari_mining_node"
+cp -f "${PWD}/applications/tari_mining_node/osx/runtime/start_tari_mining_node.sh" "${distDir}/runtime/start_tari_mining_node.sh"
+cp -f "${PWD}/target/release/tari_mining_node" "${distDir}/runtime/tari_mining_node"
+
+# Merge Mining Proxy
+cp -f -P "${PWD}/applications/tari_merge_mining_proxy/osx/start_tari_merge_mining_proxy" "${distDir}/start_tari_merge_mining_proxy"
+cp -f -P "${PWD}/applications/tari_merge_mining_proxy/osx/start_xmrig" "${distDir}/start_xmrig"
+cp -f "${PWD}/applications/tari_merge_mining_proxy/osx/runtime/start_tari_merge_mining_proxy.sh" "${distDir}/runtime/start_tari_merge_mining_proxy.sh"
+cp -f "${PWD}/applications/tari_merge_mining_proxy/osx/runtime/start_xmrig.sh" "${distDir}/runtime/start_xmrig.sh"
+cp -f "${PWD}/target/release/tari_merge_mining_proxy" "${distDir}/runtime/tari_merge_mining_proxy"
+
+# 3rd party install
+if [ "$osname" == "osx" ]; then
+cp -f "${PWD}/buildtools/install_xmrig.sh" "${distDir}/runtime/install_xmrig.sh"
+cp -f "${PWD}/buildtools/get_xmrig_osx.ps1" "${distDir}/runtime/get_xmrig_osx.ps1"
```

Comment:
Are we using somebody else powershell script? I am not sure how well this would go over with Nix users that have bash already installed?

Reply:
The powershell scripts are our own.

Suggestions?

Reply:
Everything else is using bash for the Nix environments, I would suggest sticking with that would be already installed and in use.

Reply:
If you can replace `install_xmrig.sh` and `get_xmrig_osx.ps1` using bash only that would be great; the problem to solve here is to get and install the latest version of `xmrig`, which the `ps1` script does.

Reply:
Used a combination of curl and jq in place of powershell in the bash script.